trunk/NDHMS/Routing/Reservoirs/Makefile

```diff
@@ -0,0 +1,83 @@
+#simple compilation test for modules in reservoir
```

Comment:
Is this a separate compilation from the full build? Or was it being used for both purposes?
Apparently it is part of the build: https://github.com/NCAR/wrf_hydro_nwm_public/pull/203/files?utf8=%E2%9C%93&diff=split&w=1#diff-3f0dba6dd03896bde4f34e302288f8e6
There are a lot of comments in here which make this confusing.
Why are FC and FFLAGS hard coded?

This needs a cleanup and a separation if there are dual purposes.

Reply:
resolved by 663c47e, I eliminated all comments and unused variables. I figured if you wnated to keep stuff for testing, it's in the history.

Reply:
I recommend leaving the testing target in the makefile (John needs to decide which is the correct one, of course) and uncomment it.  Remove test from the all: target so that it isn't built under normal build conditions. This will allow tests to be built and executed manually if desired (these are some basic unit like tests of the classes...)

Reply:
Thanks for cleaning. I am adding the testing back in for their own separate compilation.